src/ripple/app/tx/impl/ApplyContext.cpp

```diff
@@ -69,4 +71,42 @@ ApplyContext::visit (std::function <void (
     view_->visit(base_, func);
 }
 
+TER
+ApplyContext::checkInvariants(TER terResult)
+{
+    if (view_->rules().enabled(featureEnforceInvariants) &&
+        app.config().ENFORCE_INVARIANTS)
```

Comment:
@JoelKatz can you comment on your thinking behind the config option. What's the use case for configurable on/off ?